packages/components/src/internal/components/files/FileAttachmentForm.tsx

```diff
@@ -61,6 +61,7 @@ interface FileAttachmentFormProps {
     templateUrl?: string;
     compact?: boolean;
     ref?: any;
+    allowOversize?: boolean;
```

Comment:
Is `allowOversize` necessary?  Can the user just not pass in sizeLimits if wanting to allow any size file?

Reply:
I kept sizeLimits because it also defines preview size limit. I assume we want to keep the preview limit in place even if we support async import?

Reply:
Yes, that's true.